xml/System.Globalization/CompareInfo.xml

```diff
@@ -1767,12 +1782,14 @@
         <Parameter Name="options" Type="System.Globalization.CompareOptions" Index="2" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="source">The string to search within.</param>
+        <param name="value">The substring to locate within <paramref name="source" />.</param>
+        <param name="options">The <see cref="T:System.Globalization.CompareOptions" /> to use during the search.</param>
+        <summary>Searches for the first occurrence of a substring within a source string.</summary>
+        <returns>The zero-based index into <paramref name="source" /> where the substring <paramref name="value" /> first appears; or -1 if <paramref name="value" /> cannot be found within <paramref name="source" />.</returns>
```

Comment:
>-1 [](start = 134, length = 2)

maybe saying negative value better than -1. this give us fixability in the future if we need to return different value.

Reply:
```suggestion
        <returns>The zero-based index into <paramref name="source" /> where the substring <paramref name="value" /> first appears; or a negative value if <paramref name="value" /> cannot be found within <paramref name="source" />.</returns>
```